lib/taurus/cli/cli.py

```diff
@@ -25,26 +25,17 @@
 import click
 import taurus
 
+from taurus.cli import common as cli_common
```

Comment:
This is a general comment, and it is just a matter of personal taste, so no agreement is required:

instead of using:

```python
from taurus.cli import common as cli_common

(...)

@cli_common.foo
def bar(...)
```

I find it a bit nicer to go for a more explicit:

```python
import taurus.cli
(...)
@taurus.cli.common.foo
def bar(...)
```